src/api/streaming.ts

```
@@ -19,7 +19,7 @@ export const STREAM_MESSAGE_TYPES = {
 export const createStreamApi = async (
   apiUrl: string,
   body: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)